### ISSUE TITLE
Transfer to cumulus

### ISSUE THREAD

**Issue by jak-pan** [galacticcouncil/hack.HydraDX-node#11]
Let's transfer or change this to latest cumulus to be able to connect as a parachain. 

Let's have a look at this and decide if we're going to do it before or after merging:
https://github.com/paritytech/cumulus/pull/207

**Comment by jak-pan** [comment]
On hold until https://github.com/paritytech/cumulus/pull/224 is merged

**Comment by jak-pan** [comment]
We can start looking at splitting runtimes and integrating cumulus after #31

**Comment by jak-pan** [comment]
https://github.com/galacticcouncil/hack.HydraDX-node/pull/53
